nodes/sorter.lua

```diff
@@ -175,21 +175,23 @@ minetest.register_node("hopper:sorter", {
 		local filter_output_direction = (dir.y == 0) and "side" or "bottom"
 
 		--- returns success? = true/false
-		local function try_send_item(output_dir, dst_pos)
+		local function try_send_item(output_dir, dst_pos, filter_items_map)
 			local dst_node = minetest.get_node(dst_pos)
 			local registered_inventories = hopper.get_registered(dst_node.name)
 
 			if registered_inventories ~= nil then
-				return hopper.send_item_to(pos, dst_pos, dst_node, registered_inventories[output_dir], filter_items)
+				return hopper.send_item_to(pos, dst_pos, dst_node, registered_inventories[output_dir], filter_items_map)
 			end
 
-			return hopper.send_item_to(pos, dst_pos, dst_node, nil, filter_items)
+			return hopper.eject_item(pos, dst_pos)
```

Comment:
This line is redundant due to L193. If you'd like to keep this one here, the function would be more appropriately named `try_send_or_eject`.

Reply:
Replaced with `return false`